tools-local/Microsoft.ML.InternalCodeAnalyzer/ContractsCheckAnalyzer.cs

```diff
@@ -196,8 +196,11 @@ private static void Analyze(SyntaxNodeAnalysisContext context)
             var containingSymbolName = methodSymbol.ContainingSymbol.ToString();
             // The "internal" version is one used by some projects that want to benefit from Contracts,
             // but for some reason cannot reference MLCore.
+            // Contract functions defined Microsoft.ML.Hidden are introduced for breaking the dependencies
+            // from CpuMath project to Microsoft.ML.Core.
             if (containingSymbolName != "Microsoft.ML.Runtime.Contracts" &&
-                containingSymbolName != "Microsoft.ML.Runtime.Internal.Contracts")
+                containingSymbolName != "Microsoft.ML.Runtime.Internal.Contracts" &&
```

Comment:
when/where is the namespace `Microsoft.ML.Runtime.Internal.Contracts` used?

Reply:
None. Let's remove it.

---
In reply to: [236792043](https://github.com/dotnet/machinelearning/pull/1724#discussion_r236792043) [](ancestors = 236792043)